my_enumerable.rb

```diff
@@ -0,0 +1,17 @@
+module Enumerable
+  def all?
+    each { |item| return false unless yield item }
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use the `each` method you created to replace the ruby each method in the `all?`, `any?`, and `filter` metods